src/sentry/apidocs/parameters.py

```diff
@@ -92,6 +84,25 @@ def member_id(description: str) -> OpenApiParameter:
         )
 
 
+class OrganizationParams:
```

Comment:
@schew2381 I moved these to OrganizationParams because this documentation is valid if you're inheriting `OrganizationEndpoint`